src/silx/gui/data/DataViews.py

```diff
@@ -1089,6 +1206,15 @@ def setData(self, data):
         )
         self.__resetZoomNextTime = False
 
+    def setAggregatedData(self, data):
+        data = self.normalizeData(data)
+        self.getWidget().addImage(
+            legend="data", data=data, resetzoom=self.__resetZoomNextTime
+        )
+        self.__resetZoomNextTime = False
```

Comment:
This method does not seems to be used.

It seems that updating `def setData(self, data):` to add an `ImageAggregatedItem` instead of calling `addImage` would make it simpler: Then, changing the aggregation mode would only be a matter of changing this item's aggregation mode